core/templates/pages/release-coordinator-page/release-coordinator-page.component.ts

```diff
@@ -191,7 +203,7 @@ export class ReleaseCoordinatorPageComponent implements OnInit {
       () => {
         this.backendApiService.deleteUserGroupAsync(userGroupId).then(
           () => {
-            this.statusMessage = 'UserGroups successfully deleted.';
+            this.statusMessage = 'User Group successfully deleted.';
```

Comment:
User group XXX successfully deleted.

(Note that "group" is not capitalized -- see my [previous comment](https://github.com/oppia/oppia/pull/20429#discussion_r1789577446). Please fix elsewhere too.)

Reply:
Right, updated.